health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/MyMicroplans.js

```
@@ -0,0 +1,50 @@
+
+
+const defaultSearchValues = {
+
+};
+
+const MyMicroplans = () => {
+  const { t } = useTranslation();
+  const [defaultValues, setDefaultValues] = useState(defaultSearchValues); // State to hold default values for search fields
+  const [config, setConfig] = useState(TabSearchconfig?.TabSearchconfig?.[0]); // initially setting first index config as default from jsonarray
+  const [tabData, setTabData] = useState(
+    TabSearchconfig?.TabSearchconfig?.map((configItem, index) => ({ key: index, label: configItem.label, active: index === 0 ? true : false }))
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)